store/mockstore/unistore/cophandler/mpp.go

```diff
@@ -327,7 +328,7 @@ func HandleMPPDAGReq(dbReader *dbreader.DBReader, req *coprocessor.Request, mppC
 	}
 	mppExec, err := builder.buildMPPExecutor(dagReq.RootExecutor)
 	if err != nil {
-		return &coprocessor.Response{OtherError: err.Error()}
+		panic("build error: " + err.Error())
```

Comment:
Why cause a panic here instead of an error?

Reply:
well, It's because our mock tiflash can not return build error correctly and directly. Solving this would take handful work, Counld we leave it be for now? >_<